datasetinsights/io/gcs.py

```diff
@@ -101,3 +109,52 @@ def download_file_from_gcs(cloud_path, local_path, filename, use_cache=True):
 
     # TODO(YC) Should run file checksum before return.
     return local_filepath
+
+
+def download_folder_from_gcs(cloud_path, local_path):
+    """Helper method to download list of files from GCS
```

Comment:
This method tries to do multiple things. Might be helpful to break it down to smaller unit of functions ant make it easier for tests.